pgoapi/location.py

```diff
@@ -27,9 +27,28 @@ def get_route(start, end, use_google=False, GMAPS_API_KEY="", walk_to_all_forts=
         else:
             d = directions_service.directions(origin, destination, mode="walking", units="metric")
         steps = d[0]['legs'][0]['steps']
-        return [(step['end_location']["lat"], step['end_location']["lng"]) for step in steps]
+        final_steps = [
```

Comment:
@sontek we are planning to really remove the way point experimental feature pretty soon, within this week. If i can ask you a favor, to remove the "spin_all_forts" feature completely and remove any code you added to compensate for the google navigation. This will make things easier. Also there is no sense in using Google Walking directions honestly, it was just there if people liked to use it. 
its less efficient 
a) walk takes much longer
b) Hatching egg distance is calculated point to point, if the bot is roaming around using google navigation in a small area it wont collect distance travelled easily. 

So for this PR you can if you chose to accept, remove the spin_all_forts feature and assume that use google will not be used. So this way we Guarantee that We will ALWAYS reach a fort if we have decided to walk to it. IT will Never be out of range.